### ISSUE TITLE
about the negative cosine proximity loss

### ISSUE THREAD

**Issue by suzhenghang** [taoyang1122/pytorch-SimSiam#2]
Hi @taoyang1122 , thanks for opening such good codes. What is the loss value at the end of training and could you share the loss curve?

**Comment by taoyang1122** [comment]
The loss would be around -9.3~-9.4. There is a loss curve in the original paper (Fig.2), you can check that.

**Comment by suzhenghang** [comment]
Thanks. I did both of unsupervised pretraining (**BatchSize512+SGD+100Epoch+CosineLR0.1+NegativeCosineProximity loss**)and linear evaluation(**BatchSize4096+LARS+100Epoch+CosineLR1.6+CrossEntropy loss**), and finally got 68.0%. 
 I found that Sycn-BatchNorm is critical. Without Sycn-BatchNorm, i could get 65.1% only.  The training curve is as follow.
![simsiam](https://user-images.githubusercontent.com/8575329/106598868-4a88a880-6593-11eb-9535-ee41ffbc8dc0.png)
![simsiam_lincls](https://user-images.githubusercontent.com/8575329/106598876-4eb4c600-6593-11eb-9213-00198c28f748.png)

**Comment by taoyang1122** [comment]
Yes, SyncBN seems to be critical. Thanks for the sharing.
